spec.txt

```diff
@@ -9188,6 +9197,28 @@ foo <a href="\*">
 ````````````````````````````````
 
 
+A block quote can prevent a line from being parsed as inline HTML,
+even though line breaks are allowed in tags:
+
+```````````````````````````````` example
+<a
+>
+.
+<p>&lt;a</p>
+<blockquote>
+</blockquote>
+````````````````````````````````
```

Comment:
It might be nice here to not have an empty block quote, but to actually have a word there? I don’t think many folks will want to write empty block quotes?